fw/config.cpp

```diff
@@ -1,58 +1,91 @@
-#define CONFIG_VERSION 6
-
+#pragma region Addresses
```

Comment:
Please do not include text editor-specific code.

Reply:
Removed.